docs/content/content-blocks.md

```diff
@@ -359,14 +384,16 @@ own titles and stylings. For example:
 source
 ^^^
 ````md
-:::{admonition,dropdown,tip} Click here!
+:::{admonition} Click here!
+:class: tip, dropdown
 Here's what's inside!
 :::
 ````
 ---
 result
 ^^^
-:::{admonition,dropdown,tip} Click here!
+:::{admonition} Click here!
+:class: tip, dropdown
```

Comment:
@choldgraf I think the use of `class` here is great and makes the recommended syntax a lot clearer.